portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,9 @@
+::root{
+    font-family:"Roboto";
```

Comment:
use the [`font` shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/font) instead of the set of `font-*` properties. Prefer shorthand properties in general: https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

Reply:
Thanks, gone ahead and made that change in commit 79425f7